portal/templates/portal/mouseflow.html

```diff
@@ -0,0 +1,9 @@
+<script type="text/javascript">
+  var _mfq = _mfq || [];
+  (function() {
+    var mf = document.createElement("script");
+    mf.type = "text/javascript"; mf.async = true;
+    mf.src = "//cdn.mouseflow.com/projects/4e6407b5-890c-4dfc-9b50-fc14eeeeb2eb.js";
```

Comment:
@mikebryant I've just spoken to @CelineBoudier and MouseFlow will ignore any incoming requests not from our production domain (i.e. http://codeforlife.education), so we won't be confused by random results coming in. It would mean that anyone wanting to run their own instance of portal with heatmaps would need to fork our repo and change the above, but for now I think we're ok to merge.